Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -552,6 +553,19 @@ def install
   def test_home
     assert_equal HOMEBREW_WWW,
                  cmd("home", {"HOMEBREW_BROWSER" => "echo"})
+
+    formula_file = CoreTap.new.formula_dir/"testball.rb"
+    formula_file.write <<-EOS.undent
+      class Testball < Formula
+        desc "Some test"
+        homepage "https://example.com/testball"
+        url "https://example.com/testball-0.1.tar.gz"
+      end
+    EOS
+
+    assert_equal Formula["testball"].homepage, cmd("home", "testball", {"HOMEBREW_BROWSER" => "echo"})
```

Comment:
Just a stylistic nit: Can you wrap this like the other `assert_equal` in this method? I think this makes it more readable. We try to stay below the 80-column limit which helps with seeing the full code in a narrow editor window without vertical scrolling.